metricbeat/module/vsphere/host/host.go

```diff
@@ -62,7 +62,7 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 func (m *MetricSet) Fetch() ([]common.MapStr, error) {
 	f := find.NewFinder(m.Client, true)
 	if f == nil {
```

Comment:
@andrewkroh Sorry, maybe my rebase went wrong. I'll correct this and remove this check. In another PR, I'll see if it's possible to use the new package view instead of finder (like [here](https://github.com/vmware/govmomi/pull/716))